src/child/containers/sidebars/favourites/Favourites.js

```diff
@@ -69,6 +73,20 @@ class Favourites extends Component {
         }
     }
 
+    onDropOutside(e, stockCode) {
+        this.props.dispatch(dragOut(stockCode, this.props.favourites.names[stockCode]));
+
+        // Unfavourite the stock from this window
+        this.props.bindings.toggleFavourite(stockCode);
+
+        const childWindow = new fin.desktop.Window(
```

Comment:
This feels quite odd - creating a new child window from the favourites container in a child window.

Feels like it should be the responsibility of the parent. Perhaps an action could be dispatched which will open the new window?